src/idiom274.fuzion

```diff
@@ -0,0 +1 @@
+y := strings.fromBytes (x.utf8 & (c -> !x.isAsciiWhiteSpace c))
```

Comment:
Looks good, just the spec says: _Create the string t from the string s,... _, there is no `t`nor `s`in your code...:-)